src/indexer.h

```diff
@@ -516,6 +516,18 @@ struct FileContents {
   FileContents(const std::string& path, const std::string& content);
 };
 
+struct FileContentsWithOffsets {
+  std::string contents_;
```

Comment:
contents

Reply:
remove trailing `_` for the two members?

Reply:
Only on this one.

Reply:
Done